tree/dataframe/test/dataframe_vary.cxx

```diff
@@ -662,84 +862,337 @@ TEST(RDFVary, VaryDisplay) // TEST instead of TEST_P because Display is single-t
    // must update this test when https://github.com/root-project/root/issues/9894 is addressed
 }
 
-TEST_P(RDFVary, DISABLED_VaryFill)
-{
+struct Jet {
+   double a, b;
+};
 
-}
+struct CustomFiller {
 
-TEST_P(RDFVary, DISABLED_VaryGraph)
-{
+   TH2D h{"", "", 10, 0, 10, 10, 0, 10};
 
-}
+   void Fill(const Jet &j) { h.Fill(j.a, j.b); }
 
-TEST_P(RDFVary, DISABLED_VaryGraphAsymmErrors)
-{
+   void Merge(const std::vector<CustomFiller *> &jets)
+   {
+      TList l;
+      for (auto *j : jets)
+         l.Add(&j->h);
+      h.Merge(&l);
+   }
+};
 
-}
+TEST_P(RDFVary, VaryCustomObject)
+{
+   // this object can be filled
+   auto h = ROOT::RDataFrame(10)
+                   .Define("Jet",
+                           [] {
+                              return Jet{1., 2.};
+                           })
+                   .Vary(
+                      "Jet",
+                      [] {
+                         return ROOT::RVec<ROOT::RVecD>{{}, {4., 5.}};
```

Comment:
we are varying the `"Jet"` column which is of type `Jet`. As variations we are returning two `RVecD`. That should not be allowed, I'm surprised it works.

Reply:
This was actually a failing test - giving a runtime error with the message - Cannot extract type_info of type CLING_UNKNOWN_TYPE_Jet. I am about to commit a passing test now.

Reply:
my comment above still applies

Reply:
yes, this looks better